cmd/posix_test.go

```diff
@@ -35,7 +36,13 @@ func newPosixTestSetup() (StorageAPI, string, error) {
 	if err != nil {
 		return nil, "", err
 	}
-	posixStorage, err := newPosix(diskPath)
+	var endpoints []*url.URL
+	// Initialize a new posix layer.
+	endpoints, err = parseStorageEndpoints([]string{diskPath})
+	if err != nil {
+		return nil, "", err
+	}
+	posixStorage, err := newPosix(endpoints[0])
```

Comment:
Actually newStorageAPI already does this, this logic is not needed there.  One particular reason one needs is when printing the local disks we loose the information that provided on command line. 

For example in current master 

```
Initializing data volume.
[01/16] /mnt/export1/minio - 218 GiB online
[02/16] /mnt/export2/minio - 218 GiB online
[03/16] minio2:/mnt/export1/minio - 218 GiB online
[04/16] minio2:/mnt/export2/minio - 218 GiB online
[05/16] minio3:/mnt/export1/minio - 218 GiB online
[06/16] minio3:/mnt/export2/minio - 218 GiB online
[07/16] minio4:/mnt/export1/minio - 218 GiB online
[08/16] minio4:/mnt/export2/minio - 218 GiB online
[09/16] minio5:/mnt/export1/minio - 218 GiB online
[10/16] minio5:/mnt/export2/minio - 218 GiB online
[11/16] minio6:/mnt/export1/minio - 218 GiB online
[12/16] minio6:/mnt/export2/minio - 218 GiB online
[13/16] minio7:/mnt/export1/minio - 218 GiB online
[14/16] minio7:/mnt/export2/minio - 218 GiB online
[15/16] minio8:/mnt/export1/minio - 218 GiB online
[16/16] minio8:/mnt/export2/minio - 218 GiB online
```

It is neat to know what was the suppliedOriginal path.